proto/gen/einride/example/freight/v1/shipment.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.17.3
+// 	protoc        (unknown)
```

Comment:
@odsod not a 100% sure why we are getting unknown here. Maybe we can have a look at this together

Reply:
It's intentional. Buf sets `(unknown)` since `buf generate` does not use `protoc` at all, except when generating the built-in languages (Python, Java, C++, etc).

Reply:
:+1: